misc/python/materialize/buildkite.py

```diff
@@ -38,7 +38,8 @@ def is_in_pull_request() -> bool:
 
 
 def is_pull_request_marker_set() -> bool:
-    return ui.env_is_truthy("BUILDKITE_PULL_REQUEST")
```

Comment:
Although I extended `env_is_truthy`, I slightly prefer to keep it more explicit here.